### PR TITLE
[Hotfix] Re-arrange the test package structure of streampark-console-service.

### PR DESCRIPTION
**Pull request by RocMarshal**
Problem Summary:

### What is changed and how it works?

Proposal: Only code-cleanup work. Re-arrange the test package structure of the streampark-console-service module.

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

## Purpose of this pull request

Re-arrange the test package structure of the streampark-console-service module.

### CODE REVIEW

**Comment by wolfboys** [comment]
Thanks for your contribution. I've seen many package name has changed, it is not under the same package name,  Can you tell me why we need to change like this?

**Comment by RocMarshal** [comment]
@wolfboys 
- Some test cases about built-in classes like ```CompletableFutureTest.java```、```RegexTest```  should not be located in the repo. Because It looks more like a usage example. However, the tested classes have already be stable.
- Some test cases about third-part dependencies like ```caffeine``` are also same as mentioned above.
- The test cases classes files would be better located in the same package which contains target tested classes to test  under the corresponding test module.  In addition, other modules' test cases package structure also follow this rule

File: streampark-common/src/test/scala/org/apache/streampark/common/util/YarnTest.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    val numYarnMaxVcores = HadoopUtils.yarnClient.getNodeReports(NodeState.RUNNING).stream.mapToInt(report => report.getCapability.getVirtualCores).max.orElse(0)
+    System.out.println(numYarnMaxVcores)
+  }
+
+  @Test def getURL(): Unit = {
+    /**
+     * 将hadoop的配置文件放到一个目录下,
+     * 在运行该类的时候加上jvm级别的参数(idea里的 vmOption ) -DHADOOP_CONF_DIR=${目录}
+     */
+    val url = YarnUtils.getRMWebAppURL
```
Comment (lvshaokang) [inline streampark-common/src/test/scala/org/apache/streampark/common/util/YarnTest.scala]: @RocMarshal Thanks for you feedback. Could you handle these non-ascaii characters by remove or translate ways?

**Review by lvshaokang — CHANGES_REQUESTED** [review]

File: streampark-console/streampark-console-service/src/test/java/com/github/benmanes/caffeine/cache/RefreshCacheTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        System.out.println(DateUtils.format(new Date(), DateUtils.fullFormat(), TimeZone.getDefault()));
+        Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                System.out.println(DateUtils.format(new Date(), DateUtils.fullFormat(), TimeZone.getDefault()));
+            }
+        }, 1000 * 10, 1000 * 10);
+
+        Thread.sleep(Long.MAX_VALUE);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/test/java/com/github/benmanes/caffeine/cache/RefreshCacheTest.java]: `Thread.sleep(Long.MAX_VALUE)` will block the thread. You can delete this testcase, see: https://github.com/apache/incubator-streampark/pull/1595

Comment (RocMarshal) [reply streampark-console/streampark-console-service/src/test/java/com/github/benmanes/caffeine/cache/RefreshCacheTest.java]: @wolfboys Thx for the comment. 
Updated. PTAL.

File: streampark-common/src/test/scala/org/apache/streampark/common/util/YarnTest.scala (hunk 1 above)
Comment (RocMarshal) [reply streampark-common/src/test/scala/org/apache/streampark/common/util/YarnTest.scala]: @lvshaokang Thx for the comment.
Updated. PTAL.

**Review by wolfboys — DISMISSED** [review]
LGTM

**Review by wolfboys — COMMENTED** [review]
the YarnTest failed, 😞 this testcase can be deleted

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**